### DISCUSSION TITLE
Create dynamic form

### DISCUSSION THREAD

**Discussion by DavidGilWeb** [filamentphp/filament#11627] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hello, in my project I have as an objective, to make a form that I create as a budget, therefore this will have dynamic data.
I have been trying this with resource, with its Edit and Create.
But when I want to treat this data with js, it is giving me problems, because for example a hidden where I store the total, gives me problems when updating.
What advice can you give me about this, should I approach it in a way other than resources or is there a solution for this?

**Comment by leandrocfe** [comment]
Please share some code you are trying to do.

*because for example a hidden where I store the total, gives me problems when updating.*

Are you using a hidden field?

**Reply by DavidGilWeb** [reply]
I'm finally setting it up by performing operations from resource, I just need to know how I can show in the view the data that is dynamically updated in the form. How could I show it, I attach my code.

------  ProjectResource ----------------
        return $form
            ->schema([
                Section::make()->schema([
                    Select::make('BuiltSqm')->id('builtSqm')
                    ->label('Metros cuadrados')
                    ->options(ProjectResource::BuiltSqmFunction($arrBSqm))->default(3)
                    ->live()
                    ->afterStateUpdated(function (Get $get, Set $set){
                        self::updateTotals($get,$set);
                    })
                    ->selectablePlaceholder(false)->native(false),
                    Hidden::make('PriceFinalAmount')->default(0),
])

------ view-data.blade -------------------
<?php $arr = json_encode($arrBSqm); ?>
<x-dynamic-component
    :component="$getFieldWrapperView()"
    :field="$field"
>
    <div x-data="{ state: $wire.entangle('{{ $getStatePath() }}') }">  
        

            <span id='ShowTotalPrice'> {{ $getRecord()!==null ? number_format($getRecord()->PriceFinalAmount, 2, ',', '.') : number_format(0, 2, ',', '.') }}</span> €
        
    </div>
</x-dynamic-component>
